### PR TITLE
merge branch of style changes

### PR DESCRIPTION
**Pull request by erikerikson**
Given my fault in the bikeshedding branch, rather than asking you to fix the style related changes, I have attempted to offer the merge of those changes here.

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-/**/node_modules
-/**/*bundle.js
+**/node_modules
```
Comment (erikerikson) [inline .eslintignore]: this is a non-style change.  the / read root in my brain and removing it should be functionally equivalent.  didn't want to slide this by without approval

File: product-catalog/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Product Catalog
```
Comment (erikerikson) [inline product-catalog/README.md]: all this product-catalog stuff can be ignored for the purposes of your branch, it comes from the merge and we reviewed it previously

File: products/lambda/product-events.js [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
-    this.kinesisEventWriter.writeKinesisEvent({
-      schema: 'com.nordstrom/product/create/1-0-0',
-      id: product.id.toString(),
-      brand: product.brand,
-      name: product.name,
-      description: product.description,
-      category: product.category,
-    }, product.id);
+    this.kinesisEventWriter.writeKinesisEvent(
+      {
```
Comment (erikerikson) [inline products/lambda/product-events.js]: additional style mod here, not trying to slide anything by

File: products/lambda/scrape-products/scrape-products.js [hunk 4]
```diff
@@ -1,49 +1,55 @@
...
+        })
+      },
+      event.productFrequency,
+    )
 
-    setTimeout(() => {
-      clearInterval(oneProductInterval);
-      callback(null, 'product scraping complete');
-    }, event.lambdaTimeout);
+    setTimeout(
```
Comment (erikerikson) [inline products/lambda/scrape-products/scrape-products.js]: additional style mod here, not trying to slide anything by

File: products/lambda/scrape-products/scrape-products.js [hunk 5]
```diff
@@ -1,49 +1,55 @@
...
-          scraped.Id.toString(),
-          scraped.Title,
-          scraped.Brand.Label,
-          scraped.category,
-          `PAGE:${scraped.ProductPageUrl}`,
-        );
-        productEvents.sendCreateEvent(product);
-      });
-    }, event.productFrequency);
+    const oneProductInterval = setInterval(
```
Comment (erikerikson) [inline products/lambda/scrape-products/scrape-products.js]: additional style mod here, not trying to slide anything by

File: products/lambda/scrape-products/kinesis-event-writer.js [hunk 6]
```diff
@@ -7,32 +7,32 @@ const impl = {
...
       Data: JSON.stringify(envelopeEvent),
       PartitionKey: partitionKey,
       StreamName: process.env.STREAM_NAME,
-    };
+    }
 
     this.kinesis.putRecord(newProductCreatedEvent, (err, ack) => {
       if (ack) {
-        console.log(`K-PUT: ${JSON.stringify(ack)}`);
+        console.log(`K-PUT: ${JSON.stringify(ack)}`)
```
Comment (erikerikson) [inline products/lambda/scrape-products/kinesis-event-writer.js]: Should have brought this up in previous review...  This breaks the standard pattern of `if (err) { ... } else { ... }` does the Kinesis response break that pattern too?

File: .eslintignore (hunk 1 above)
Comment (gwsii) [inline .eslintignore]: s/b fine

File: products/lambda/scrape-products/kinesis-event-writer.js (hunk 6 above)
Comment (gwsii) [inline products/lambda/scrape-products/kinesis-event-writer.js]: We can reverse the tests to be more consistent, sure.

**Comment by gwsii** [comment]
I'll leave it to you to delete the branch in case you want to push more to it.

**Comment by erikerikson** [comment]
Nope, thanks!

**Pull request merged**